k8s/upgrades/0.9.0-1.0.0/pre-upgrade.sh

```diff
@@ -295,7 +295,7 @@ for spc_name in `echo $spc_list | tr ":" " "`; do
     fi
 done
 
-ds_name=$(kubectl get pod -n $ns -l openebs.io/component-name=ndm \
+ds_name=$(kubectl get pod -n $ns -l component=ndm \
```

Comment:
@sdelrio thanks for the PR, `openebs.io/component` label on ndm daemon set pods is common for both way of installation of OpenEBS (i.e via operator yaml and helm installation)

Reply:
Could it be that my instalation kept labels from an old version? I think I've been doing upgrades since v0.8

```
$ kubectl get pods -n openebs --show-labels | grep ndm
openebs-ndm-k92p5                              1/1     Running   0          22d   app=openebs,component=ndm,controller-revision-hash=6985cfff56,openebs.io/version=0.9.0,pod-template-generation=3,release=openebs
openebs-ndm-n9qjt                              1/1     Running   5          29d   app=openebs,component=ndm,controller-revision-hash=6985cfff56,openebs.io/version=0.9.0,pod-template-generation=3,release=openebs
```
```
$ kubectl get pods -n openebs -l openebs.io/component-name=ndm|grep ndm
No resources found.
```
```
$ kubectl get pods -n openebs -l component=ndm  |grep ndm
openebs-ndm-k92p5   1/1     Running   0          22d
openebs-ndm-n9qjt   1/1     Running   5          29d
```